src/uwtools/tests/config/test_tools.py

```diff
@@ -351,14 +351,14 @@ def test_realize_config_single_dereference(capsys, tmp_path):
 
 
 def test_realize_config_supp_bad_format(tmp_path):
-    with raises(ValueError) as e:
-        path = tmp_path / "a.yaml"
-        supplemental_path = tmp_path / "b.clj"
-        msg = f"Cannot deduce format of '{supplemental_path}' from unknown extension 'clj'"
-        with writable(path) as f:
-            yaml.dump({"1": "a", "2": "{{ deref }}", "3": "{{ temporalis }}", "deref": "b"}, f)
-        with writable(supplemental_path) as f:
-            yaml.dump({"2": "b", "temporalis": "c"}, f)
+    path = tmp_path / "a.yaml"
+    supplemental_path = tmp_path / "b.clj"
+    msg = f"Cannot deduce format of '{supplemental_path}' from unknown extension 'clj'"
+    with writable(path) as f:
+        yaml.dump({"1": "a", "2": "{{ deref }}", "3": "{{ temporalis }}", "deref": "b"}, f)
+    with writable(supplemental_path) as f:
+        yaml.dump({"2": "b", "temporalis": "c"}, f)
+    with raises(UWError) as e:
```

Comment:
Moved some logic out of the `with` block that didn't need to be there, and expect a `UWError` instead of more generic `ValueError`.